platform/src/test/java/platformstepdefinition/PF_Header.java

```diff
@@ -0,0 +1,24 @@
+package platformstepdefinition;
+
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+
+import base.NewBaseClass;
+import cucumber.api.java.en.Given;
+import cucumber.api.junit.Cucumber;
+import platform.pageobjects.Header.Header;
+
+@RunWith(Cucumber.class)
+public class PF_Header extends NewBaseClass {
+
+	public WebDriver driver;
+
+	@Given("^Navigate to URL and open Header (.+)$")
+	public void navigate_to_url_(String browser) throws Throwable {
+		driver = openbrowser(browser);
+		Header headerobj = new Header(driver);
+		headerobj.headerAndSubMenuLinksVisibilityChecking();
```

Comment:
Refactor method name to verb-noun format
Ex: checkHeaderAndSubMenuLinksVisibility();

Reply:
Done Changes as requested

Reply:
Please rearrange method name in verb noun format
Ex: headerAndSubMenuLinksVisibilityChecking(); becomes checkHeaderAndSubMenuLinksVisibility()

It is helpful for global team to understand

Reply:
Done Changes as requested